.github/workflows/test-smokes.yml

```diff
@@ -283,8 +283,17 @@ jobs:
           echo "Pull Request URL - ${{ steps.cpr.outputs.pull-request-url }}"
           echo "Pull Request Action Performed - ${{ steps.cpr.outputs.pull-request-operation }}"
 
-      - uses: actions/upload-artifact@v3
+      - name: Upload test timing file
```

Comment:
Does this belong on his PR?

Reply:
It is just setting the name on the workflow step. I could have done it outside, just did it there while adding an upload of the playwright report.